src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMBuilder.java

```diff
@@ -127,7 +127,7 @@ public static UriTemplate checkoutUriTemplate(@CheckForNull Item context,
                     )
             );
             if (credentials instanceof SSHUserPrivateKey) {
-                return UriTemplate.buildFromTemplate("ssh://" + sshRemote)
```

Comment:
Could properly just get await with returning `sshRemote`? 😆 
I wonder in the return below would be better to return: `http_url_to_repo` that we could get from the project/group api anyway?

Reply:
I think in this we are saving an API call. This is not a primary concern now. I think we can come back to this later. I have raised a ticket - https://issues.jenkins-ci.org/browse/JENKINS-58730

Reply:
Well, we already have the data, we just need to pass as you have passed sshRemote :)

Reply:
Fixed in cd28d1a